fixtures/cameo/root-par-6.json

```diff
@@ -0,0 +1,753 @@
+          "type": "ColorPreset",
+          "comment": "Yellow"
+        },
+        {
+          "dmxRange": [38, 45],
+          "type": "ColorPreset",
+          "comment": "Green"
+        },
+        {
+          "dmxRange": [46, 53],
+          "type": "ColorPreset",
+          "comment": "Turquoise"
+        },
+        {
+          "dmxRange": [54, 61],
+          "type": "ColorPreset",
+          "comment": "Cyan"
+        },
+        {
+          "dmxRange": [62, 69],
+          "type": "ColorPreset",
+          "comment": "Blue"
+        },
+        {
+          "dmxRange": [70, 77],
+          "type": "ColorPreset",
+          "comment": "Lavender"
+        },
+        {
+          "dmxRange": [78, 85],
+          "type": "ColorPreset",
+          "comment": "Mauve"
+        },
+        {
+          "dmxRange": [86, 93],
+          "type": "ColorPreset",
+          "comment": "Magenta"
+        },
+        {
+          "dmxRange": [94, 101],
+          "type": "ColorPreset",
+          "comment": "Pink"
+        },
+        {
+          "dmxRange": [102, 109],
+          "type": "ColorPreset",
+          "comment": "Warm White"
+        },
+        {
+          "dmxRange": [110, 117],
+          "type": "ColorPreset",
+          "comment": "White"
+        },
+        {
+          "dmxRange": [118, 125],
+          "type": "ColorPreset",
+          "comment": "Cold White"
+        },
+        {
+          "dmxRange": [126, 201],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [202, 207],
+          "type": "ColorPreset",
+          "comment": "User Color_1"
+        },
+        {
+          "dmxRange": [208, 213],
+          "type": "ColorPreset",
+          "comment": "User Color_2"
+        },
+        {
+          "dmxRange": [214, 219],
+          "type": "ColorPreset",
+          "comment": "User Color_3"
+        },
+        {
+          "dmxRange": [220, 225],
+          "type": "ColorPreset",
+          "comment": "User Color_4"
+        },
+        {
+          "dmxRange": [226, 255],
+          "type": "ColorPreset",
+          "comment": "UV"
+        }
+      ]
+    },
+    "Amber": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "UV": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "UV"
+      }
+    },
+    "Strobe 2": {
+      "name": "Strobe",
+      "capabilities": [
+        {
+          "dmxRange": [0, 5],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [6, 255],
+          "type": "StrobeSpeed",
+          "speedStart": "1Hz",
+          "speedEnd": "20Hz"
+        }
+      ]
+    },
+    "Dimmer 4": {
+      "name": "Dimmer",
+      "capability": {
+        "type": "Intensity",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Dimmer fein": {
+      "capability": {
+        "type": "Intensity"
+      }
+    }
+  },
```

Comment:
This part is just straight up confusing to me.
The DMX Table classifies it as follows:
![image](https://user-images.githubusercontent.com/38377070/122136614-1e373f80-ce43-11eb-8aca-5e019755d30a.png)
So its 1 Group using 2 Seperate Channels but both for Dimming? Maybe twice the granularity?
Maybe @FloEdelmann, you have more experience with this?

In any case this still needs some changes!

Reply:
It seems that this a channel for 16bit dimming:

"Sixteen-bit dimming offers the user the ability to easily control LED dimming just like you would control fine pan and fine tilt. As always, there are 255 steps of control in 8 bit (standard) dimming control. With the fine control offered in 16-bit mode, there are 255 additional steps between each step of the standard 8-bit dimmer channel. This additional level of control allows the fixture to act much more fluid in dimming, especially at the low end where it has always been problematic."

Basically, always using the "Dimmer fine" should be the preferred way, right?

Reply:
Two channels with 256 values each (0 to 255) can be combined to one logical channel with 65536 values (0 to 65535). This is solved with *Fine channels* in OFL: https://github.com/OpenLightingProject/open-fixture-library/blob/master/docs/fixture-format.md#fine-channels

Fixed in [`366b193` (#1903)](https://github.com/OpenLightingProject/open-fixture-library/pull/1903/commits/366b19384328fc967289c9d02df73dd22ad6da64).